src/Parallel-Solvers/Parallel-Explicit/SGH_Solver/include/FEA_Module_SGH.h

```diff
@@ -678,6 +678,7 @@ class FEA_Module_SGH : public FEA_Module
 
     // Dual Views of the corner struct variables
     DViewCArrayKokkos<double> corner_force;
+    DCArrayKokkos<double> corner_external_force;
```

Comment:
What is the corner_external_force?

Reply:
Stores pre assembled contributions from external loading conditions so we can use FOR_ALL. Separate from corner force because it doesn't contribute to specific internal energy rate like stress does.